content/develop/data-types/probabilistic/configuration.md

```diff
@@ -0,0 +1,166 @@
+
+The following table summarizes which Cuckoo filter configuration parameters can be set at run-time, and compatibility with Redis Software and Redis Cloud.
+
+| Old parameter name | New parameter name                          | Run-time           | Redis<br />Software | Redis<br />Cloud |
+| :-------           | :-------                                    | :-------           | :-------              | :-------         |
+|                    | [cf-bucket-size](#cf-bucket-size)           | :white_check_mark: |||
+|                    | [cf-initial-size](#cf-initial-size)         | :white_check_mark: |||
+|                    | [cf-expansion-factor](#cf-expansion-factor) | :white_check_mark: |||
+| CF_MAX_EXPANSIONS  | [cf-max-expansions](#cf-max-expansions)     | :white_check_mark: | <span title="Supported">&#x2705; Supported</span><br /><span><br /></span> | <span title="Supported">&#x2705; Flexible & Annual</span><br /><span title="Not supported"><nobr>&#x274c; Free & Fixed</nobr></span> |
+|                    | [cf-max-iterations](#cf-max-iterations)     | :white_check_mark: |||
+
+{{< warning >}}
+A filter should always be sized for the expected capacity and the desired error rate.
+Using the `INSERT` family commands with the default values should be used in cases where many small filters exist and the expectation is most will remain at around the default sizes.
+Not optimizing a filter for its intended use will result in degradation of performance and memory efficiency.
+{{< /warning >}}
+
+## Default parameters for Bloom filters
+
+### bf-error-rate
+
+Default error ratio for Bloom filters.
+
+Type: double
+
+Valid range: `[0 .. 1]`; consider limiting the range to `[0 .. 0.25]`.
+
+Default: `0.01`
+
+### bf-expansion-factor
+
+Added in v8.0.0.
+
+Expansion factor for Bloom filters.
+
+Type: integer
+
+Valid range: `[0 .. 32768]`.
+
+Default: `2`
+
+### bf-initial-size
+
+Initial capacity for Bloom filters.
+
+Type: integer
+
+Valid range: `[1 .. 1GB]`
+
+Default: `100`
+
+## Default parameters for Cuckoo filters
+
+### cf-bucket-size
+
+Added in v8.0.0.
+
+The number of items in each Cuckoo filter bucket.
+
+Type: integer
+
+Valid range: `[1 .. 255]`
+
+Default: `2`
+
+### cf-initial-size
+
+Added in v8.0.0.
+
+Cuckoo filter initial capacity.
+
+Type: integer
+
+Valid range: `[2*CF_BUCKET_SIZE .. 1GB]`
+
+Default: `1024`
+
+### cf-expansion-factor
+
+Added in v8.0.0.
+
+Expansion factor for Cuckoo filters.
+
+Type: integer
+
+Valid range: `[0 .. 32768]`
+
+Default: `1`
+
+### cf-max-expansions
+
+The maximum number of expansions for Cuckoo filters.
+
+Type: integer
+
+Valid range: `[1 .. 65535]`
+
+Default: `32`
+
+### cf-max-iterations
+
+Added in v8.0.0
+
+The maximum number of iterations for Cuckoo filters.
+
+Type: integer
+
+Valid range: `[1 .. 65535]`
+
+Default: `20`
+
+## Setting configuration parameters on module load (deprecated)
```

Comment:
We need to mention that starting with v8.0 the probabilistic configuration parameters will be persisted in the same way as Redis configuration parameters (e.g., with `CONFIG REWRITE` for CE).